### ISSUE TITLE
[Enhancement Request] Please consider Advanced Rocketry Integration

### ISSUE THREAD

**Issue by DVorriose** [CD4017BE/VerticallyStackedDimensions#10]
Advanced Rocketry uses a single dimension for "Space," but has some sort of reference (not sure where in code) to define where in space it is ("over/orbiting" which planet or moon). 

Would you consider adding an option to this reference so we could config a maximum height to orbit?

**Comment by Krutoy242** [comment]
Yeah, would be awesome to jump out of Space station right onto planet.
